src/azure-cli/azure/cli/command_modules/network/tests/latest/test_network_commands.py

```diff
@@ -3355,8 +3355,8 @@ def test_network_lb(self, resource_group):
 
         # test lb create with min params (new ip)
         self.cmd('network lb create -n {lb}1 -g {rg}', checks=[
-            self.check('loadBalancer.frontendIPConfigurations[0].properties.privateIPAllocationMethod', 'Dynamic'),
-            self.check('loadBalancer.frontendIPConfigurations[0].resourceGroup', '{rg}')
+            self.check('frontendIPConfigurations[0].privateIPAllocationMethod', 'Dynamic'),
```

Comment:
I have a small concern that why "loadBalancer" and "properties" are disappeared here?

Reply:
Because the template was called to create the resource, it is now removed.